src/main/scala/cromwell/engine/package.scala

```diff
@@ -29,6 +29,10 @@ import scala.util.{Failure, Success, Try}
  * Internally, this package is built on top of [[cromwell.binding]].
  */
 package object engine {
+
+  type Hash = String
+  type FileHasher = WdlFile => Hash
```

Comment:
Yes for `Hash`, for types that are actually functions I like keeping them as `type`s because it feels more natural to just return the function instead of a case class with something like `FileHasher({ wdlFile: WdlFile => ... })` but if we really want strong types everywhere then so be it :smile: